edx_lint/tamper_evident.py

```diff
@@ -26,21 +26,21 @@ def write(self, text, hashline=b"# {}"):
         be changed to accommodate different file syntaxes.
 
         Arguments:
-            text (byte string): the contents of the file to write.
+            text (text string): the contents of the file to write.
```

Comment:
Let's get off on the right Python 3 foot, and say "unicode string", since that's what it is.